config/config.exs

```diff
@@ -44,6 +44,8 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:remote_ip, :request_id]
 
+config :logster, :filter_parameters, ["password", "secret", "token"]
```

Comment:
In the documentation, this appears to be an example configuration rather than a recommended default. Does this align with any sensitive parameters sent to our endpoints?